src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -49,6 +49,14 @@ unsafe static int Execute(int argc, char** argv, NativeBootstrapperContext* cont
             bootstrapperContext.TargetFramework = new FrameworkName(FrameworkNames.LongNames.DnxCore, new Version(5, 0));
             bootstrapperContext.HandleExceptions = context->HandleExceptions;
 
+            Logger.TraceInformation($"Operating System: {bootstrapperContext.OperatingSystem}");
```

Comment:
This is superhelpful to see what actually was passed from native parts without attaching the debugger (and on Mac/Linux you cannnot debug managed (CoreClr)). At the moment this information does not reach the host and when it does it will be broken down to environment and appenvironment etc. I want to leave it here at least for now.